modules/ROOT/partials/configuration/details_serialized_state.adoc

```diff
@@ -13,10 +13,10 @@ The default value for this option is `+inherited+`.
 
 *Type:* `+String+`
 
-*Possible values:* `+inherited+`, `+collapsed+`, `+expanded+`
-
 *Default value:* `+inherited+`
 
+*Possible values:* `+inherited+`, `+collapsed+`, `+expanded+`
```

Comment:
Based on [this](https://github.com/tinymce/tinymce/blob/d739cd9dd0a2aad39970759d4e2fc94c9217d177/modules/tinymce/src/core/main/ts/api/Options.ts#L874) these should be wrapped in `' '`
```suggestion
*Possible values:* `'+inherited+'`, `'+collapsed+'`, `'+expanded+'`
```